### DISCUSSION TITLE
AMD Kipudrv Driver ?

### DISCUSSION THREAD

**Discussion by Dimitrios31337** [GSDragoon/RadeonSoftwareSlimmer#94] [General]
Hello,

awesome Tool, thanks ! But i sadly cannot find any information about this "new" Entry here with the latest Drivers.

![AMD Driver unknown Entry](https://github.com/GSDragoon/RadeonSoftwareSlimmer/assets/35952386/3b1d17f9-78be-4de7-99c9-3dc39210dc23)

Google could not help me, so i hope someone here has some better insight what exactly it is, thank you.

Best regards

**Comment by danielstreba** [comment]
It's related to this: "Ryzen AI Software supports the AMD Ryzen™ 7940HS, 7840HS, 7640HS, 7840U and 7640U processors running Windows 11."

https://ryzenai.docs.amd.com/en/latest/inst.html

**Reply by Dimitrios31337** [reply]
Awesome, thanks for the Information and Source Daniel.
